econml/sklearn_extensions/model_selection.py

```diff
@@ -599,9 +600,20 @@ def get_selector(input, is_discrete, *, random_state=None, cv=None, wrapper=Grid
     named_models = {
         'linear': (LogisticRegressionCV(random_state=random_state, cv=cv) if is_discrete
                    else WeightedLassoCVWrapper(random_state=random_state, cv=cv)),
+        'poly': ([make_pipeline([PolynomialFeatures(d),
```

Comment:
Seems there could potentially be some weird interactions if a user passes 'poly' and also has something like PolynomialFeatures as an X featurizer. 

Not sure if we should do anything to protect or warn against this though

Reply:
I don't think this is a problem: featurization affects the final model but not the first stage models, whereas this is for selecting a good first stage model.